<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeDataWindow</class>
 <widget class="QDialog" name="ChangeDataWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Изменение данных в БД</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/rec/img/ico.ico</normaloff>:/rec/img/ico.ico</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="addDataTab">
      <property name="font">
       <font>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <attribute name="title">
       <string>Внесение данных в БД</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelTableName">
         <property name="font">
          <font>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="text">
          <string>Выбор таблицы для добавления данных:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="pushData">
         <property name="font">
          <font>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover { background-color: rgb(186, 189, 182) }
QPushButton:pressed { background-color: rgb(85, 87, 83) }</string>
         </property>
         <property name="text">
          <string>Сохранить в БД</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="LoadFileBtn">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover { background-color: rgb(186, 189, 182) }
QPushButton:pressed { background-color: rgb(85, 87, 83) }</string>
         </property>
         <property name="text">
          <string>Загрузить данные из файла</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QTableWidget" name="addTableWidget"/>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QComboBox" name="listNameTable"/>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="addRowBtn">
         <property name="font">
          <font>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover { background-color: rgb(186, 189, 182) }
QPushButton:pressed { background-color: rgb(85, 87, 83) }</string>
         </property>
         <property name="text">
          <string>Добавить строку в таблицу</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover { background-color: rgb(186, 189, 182) }
QPushButton:pressed { background-color: rgb(85, 87, 83) }</string>
         </property>
         <property name="text">
          <string>Удалить выделенные элементы</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="createDelTab">
      <property name="font">
       <font>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <attribute name="title">
       <string>Очистка/Удаление таблицы</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="labelTableName_2">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Выбор таблицы для удаления данных:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="listNameTablesDelCrt">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearTblBtn">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:hover { background-color: rgb(186, 189, 182) }
QPushButton:pressed { background-color: rgb(85, 87, 83) }</string>
           </property>
           <property name="text">
            <string>Очистить таблицу</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delTblBtn">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:hover { background-color: rgb(186, 189, 182) }
QPushButton:pressed { background-color: rgb(85, 87, 83) }</string>
           </property>
           <property name="text">
            <string>Удалить таблицу</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <widget class="QGraphicsView" name="graphicsView_2">
         <property name="styleSheet">
          <string notr="true">border-image: url(:/rec/img/workTable.jpg)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="labelDel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>Поле ввода параметра для  удаления из базы данных:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineDelEdit">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelNameTbl">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>Таблица для удаления данных:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="selectTableBox">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delBtn">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:hover { background-color: rgb(186, 189, 182) }
QPushButton:pressed { background-color: rgb(85, 87, 83) }</string>
           </property>
           <property name="text">
            <string>Удалить запись в базе данных</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QGraphicsView" name="graphicsView">
         <property name="maximumSize">
          <size>
           <width>973</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/rec/img/database.drawio.png);</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>60</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
